TileFormats/GeometryData/README.md

```diff
@@ -0,0 +1,108 @@
+# Geometry Data
+
+## Contributors
+
+* Dan Bagnell, [@bagnell](https://github.com/bagnell)
+* Patrick Cozzi, [@pjcozzi](https://twitter.com/pjcozzi)
+* Sean Lilley, [@lilleyse](https://github.com/lilleyse)
+
+## Overview
+
+The _Geometry_ tile format allows streaming of geometry datasets including boxes, cylinders, ellipsoids, and spheres.
+
+Each box, cylinder, ellipsoid, and sphere is a _feature_ in the 3D Tiles specification language. 
+
+## Layout
+
+A tile is composed of two sections: a header immediately followed by a body.
+
+## Header
+
+The 28-byte header contains the following fields:
+
+| Field name | Data type | Description |
+| --- | --- | --- |
+| `magic` | 4-byte ANSI string | `"geom"`. This can be used to identify the arraybuffer as a Geometry tile. |
+| `version` | `uint32` | The version of the Geometry Data format. It is currently `1`. |
+| `byteLength` | `uint32` | The length of the entire tile, including the header, in bytes. |
+| `featureTableJSONByteLength` | `uint32` | The length of the feature table JSON section in bytes. |
+| `featureTableBinaryByteLength` | `uint32` | The length of the feature table binary section in bytes. If `featureTableJSONByteLength` is zero, this will also be zero. |
+| `batchTableJSONByteLength` | `uint32` | The length of the batch table JSON section in bytes. Zero indicates that there is no batch table. |
+| `batchTableBinaryByteLength` | `uint32` | The length of the batch table binary section in bytes. If `batchTableJSONByteLength` is zero, this will also be zero. | 
+
+If `featureTableJSONByteLength` equals zero, the tile does not need to be rendered.
+
+The body section immediately follows the header section, and is composed, in order, of two fields: `Feature Table` and  `Batch Table`.
+
+Code for reading the header can be found in
```

Comment:
Throughout just like to Cesium's master branch - later this can be a tag so it never breaks.  The PR will be merged _soon_.